test/MonitoringTest.py

```diff
@@ -0,0 +1,10 @@
+"""Test that monitoring tools are working"""
+from MediawikiTest import MediawikiBase
+
+class GrafanaTest(MediawikiBase):
```

Comment:
Suggestion for another test: Request to prometheus server.

Reply:
did that